src/stactools/hotosm/maxar/stac.py

```diff
@@ -0,0 +1,141 @@
+"""Create STAC records for HOT OSM from Maxar's public catalog."""
+
+import datetime as dt
+
+from pystac import (
+    Catalog,
+    Collection,
+    Extent,
+    Item,
+    Link,
+    MediaType,
+    RelType,
+    SpatialExtent,
+    TemporalExtent,
+)
+from pystac.extensions.item_assets import ItemAssetDefinition
+from pystac.extensions.render import Render, RenderExtension
+
+from stactools.hotosm.stac_common import add_alternate_assets
+
+COLLECTION_ID = "maxar-opendata"
+COLLECTION_DESCRIPTION = (
+    "Maxar Open Data Catalog, formatted for Humanitarian OpenStreetMap "
+    "Team's OpenAerialMap project"
+)
+
+
+def create_collection(catalog: Catalog, event_dates: list[dt.datetime]) -> Collection:
+    """Rewrite Maxar root Catalog into a Collection for HOT OAM."""
+    catalog_as_dict = catalog.to_dict()
+
+    collection = Collection(
+        id=COLLECTION_ID,
+        title=catalog_as_dict["description"],
+        description=COLLECTION_DESCRIPTION,
+        extent=Extent(
+            spatial=SpatialExtent([[-180.0, -90.0, 180.0, 90.0]]),
+            temporal=TemporalExtent([min(event_dates), None]),
+        ),
+        license=catalog_as_dict["license"],
+    )
+
+    if (catalog_self_link := catalog.get_self_href()) is not None:
```

Comment:
It pains me to suggest this more "Pythonic" way of rejecting empty strings too?

```suggestion
    if catalog_self_link := catalog.get_self_href():
```